docs/compute/monitoring-resources.md

```diff
@@ -37,7 +37,7 @@ You have sucessfully loaded slurmtools, a collection of functions
 
```

Comment:
In the `Step 4: Get some metrics!` section there are proposed items in bold e.g. `How many Service Units (core hours) have I used?` can you turn these into Markdown subsections?

Reply:
The subsections you created are not correct. Subsections in that section need to have the right amount of `#`

Reply:
Also, you missed `What jobs have I run over the past few days?`